{"name": "test_get_invalid_post[101]", "status": "broken", "statusDetails": {"message": "KeyError: 'api_base_url2'", "trace": "config = {'allure': {'build_name': 'Local Test Run', 'build_order': '1', 'executor_name': 'Local Machine', 'report_name': 'API ...'}, 'default_headers': {'Accept': 'application/json', 'Content-Type': 'application/json'}, 'environment': 'Local', ...}\ninvalid_id = 101\n\n    @allure.feature(\"Negative Test - GET /posts/{invalid_id}\")\n    @pytest.mark.parametrize(\"invalid_id\", [0, 101, \"no-id\"])\n    def test_get_invalid_post(config, invalid_id):\n>       client = PostClient(config[\"api_base_url2\"])\nE       KeyError: 'api_base_url2'\n\ntests\\test_posts_api.py:34: KeyError"}, "parameters": [{"name": "invalid_id", "value": "101"}], "start": 1747038360203, "stop": 1747038360204, "uuid": "8f00cd19-c25f-4260-b073-8ca923c6676d", "historyId": "b6f080bcb39cf28028776cb1a248edbb", "testCaseId": "a8ffa799a365376d27c1abac1a0c2678", "fullName": "tests.test_posts_api#test_get_invalid_post", "labels": [{"name": "feature", "value": "Negative Test - GET /posts/{invalid_id}"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_posts_api"}, {"name": "host", "value": "PLWRS-PF43AQAR"}, {"name": "thread", "value": "10020-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_posts_api"}]}